Backend/8taskYP/src/ThirdServlet.java

```diff
@@ -0,0 +1,34 @@
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import com.google.gson.*;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ThirdServlet extends HttpServlet {
```

Comment:
That servlet didn't mentioned in the web descriptor.
So POST check definitely will not work.